Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -268,21 +268,19 @@ public bool MakeTraitor(ICommonSession traitor, bool giveUplink = true, bool giv
         _roleSystem.MindAddRole(mindId, new TraitorRoleComponent
         {
             PrototypeId = traitorRule.TraitorPrototypeId
-        });
-        // Assign briefing
+        }, mind);
+        // Assign briefing and greeting sound
         _roleSystem.MindAddRole(mindId, new RoleBriefingComponent
         {
             Briefing = briefing
-        });
+        }, mind);
+        _roleSystem.MindAddRole(mindId, new RoleSoundComponent
+        {
+            Sound = traitorRule.GreetSoundNotification
+        }, mind);
```

Comment:
Should probably put these in some helper method or something at some point.

Reply:
like for adding array of components or just for greeting

Reply:
@metalgearsloth pro

Reply:
The greeting